pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/secretmanagersecretversion/_http.log

```diff
@@ -63,36 +65,6 @@ X-Xss-Protection: 0
 
 ---
 
-GET https://secretmanager.googleapis.com/v1/projects/${projectId}/secrets/secretmanagersecretversion-dep-${uniqueId}?alt=json
```

Comment:
seems that the previous TF-based approach has an additional REST call to get the secret.